src/applications/vaos/express-care/index.jsx

```diff
@@ -50,11 +51,14 @@ function NewExpressCareRequestSection({
 
   useEffect(
     () => {
-      if (windowsStatus === FETCH_STATUS.succeeded && !allowRequests) {
+      if (
+        !useNewFlow ||
```

Comment:
I'm not sure why we weren't doing this before, we need to make sure the EC flow doesn't work when navigating directly to it while the flag is off.